sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -35,16 +35,17 @@
 
   @Override
   public void start() {
-    options.setEnableExternalConfiguration(true);
-    options.setSentryClientName(BuildConfig.SENTRY_LOGBACK_SDK_NAME);
-    options.setSdkVersion(createSdkVersion(options));
-    Optional.ofNullable(transport).ifPresent(options::setTransport);
-    try {
-      Sentry.init(options);
-    } catch (IllegalArgumentException e) {
-      addWarn("Failed to init Sentry during appender initialization: " + e.getMessage());
+    if (!Sentry.isEnabled()) {
+      options.setEnableExternalConfiguration(true);
```

Comment:
just a question, why does the log4j use the options callback to set values and logback creates its own, this is due to DI?